services/external-dns/metadata.yaml

```diff
@@ -13,3 +13,4 @@ overview: |-
LjYxLS44NjUgMS44MzktLjU1NiAzLjk1Mi0uNDg1IDQuMzg5IDIuMzg2LjEyNS44MjEuMTY1IDEuNjIyLS4wMDcgMi40MzgtLjQ5NiAyLjM1Ni0yLjAxNyAzLjM3My00LjM5MyAyLjk1LS40NDctLjA3OS0uNjA5LS4yNi0uNTk4LS43MTNsLjAwOC0zLjE5bTkwLjA1Ni0xLjYwMWMtLjY5NyAwLTEuMzk0LS4wMTktMi4wOS4wMDYtLjQ3Ni4wMTgtLjU2NC0uMTkyLS40OS0uNjA5LjItMS4xNDIgMS4zOTUtMi4xMjMgMi42MTgtMi4xMjcgMS4yOS0uMDA1IDIuMjk1Ljg4MyAyLjU0IDIuMTU0LjA5NC40ODctLjA2LjU4OS0uNDg3LjU4bC0yLjA5LS4wMDQiLz48cGF0aCBkPSJNMTMxLjg0NCAxODcuNTUxYy0uNjk1LS4wMDEtMS4zOS0uMDA0LTIuMDg1LjAwMS0uMzIyLjAwMi0uNDktLjA2NC0uNDUtLjQ2Ny4xMjItMS4yMDUgMS4yNC0yLjE5NiAyLjU3Ni0yLjIzIDEuMjk3LS4wMzIgMi4zNzUuODU2IDIuNTg5IDIuMDY4LjA4OC41MDItLjA1OC42NTMtLjU0NC42MzVsLTIuMDg1LS4wMDdtNDMuODIxIDBjLS43MjkgMC0xLjQ1Ny0uMDA1LTIuMTg2LjAwMi0uNDA2LjAwNC0uNTE3LS4xNDItLjQyNy0uNTYuMjY0LTEuMjE1IDEuMzQzLTIuMTI4IDIuNTk4LTIuMTQyIDEuMzAzLS4wMTUgMi4yNzYuNzU3IDIuNTY5IDIuMDA3LjExNS40OTQuMDI4LjczOC0uNTUxLjcwMy0uNjY2LS4wNC0xLjMzNS0uMDA5LTIuMDAzLS4wMU0xNTAuMyA3Ni4wNjNjLTEuNzY5LS4wMTItMy4yNTYtMS41MTYtMy4yMy0zLjI2OC4wMjYtMS43NzMgMS41MjMtMy4yMTQgMy4zMDYtMy4xODYgMS43MjQuMDI3IDMuMiAxLjUxMSAzLjIgMy4yMTcgMCAxLjc0MS0xLjUyNiAzLjI0OS0zLjI3NiAzLjIzN203NS42MjQgNzUuMzk4Yy0xLjc2MS0uMDAxLTMuMjY2LTEuNTA0LTMuMjUyLTMuMjQ4LjAxNC0xLjc1OSAxLjUxNi0zLjIyMSAzLjI5My0zLjIwNmEzLjI3IDMuMjcgMCAwIDEgMy4xOTcgMy4yMDJjLjAxNCAxLjc2My0xLjQ3IDMuMjUzLTMuMjM4IDMuMjUyIi8+PHBhdGggZD0iTTIxNS44MTUgMTA3LjI4M2EzLjIgMy4yIDAgMCAxIDMuMTY0IDMuMjkyYy0uMDM2IDEuNzM5LTEuNDk0IDMuMTU2LTMuMjM4IDMuMTQ5LTEuODI5LS4wMDgtMy4yMzUtMS40ODgtMy4xOTEtMy4zNTkuMDQxLTEuNzUxIDEuNDc4LTMuMTA4IDMuMjY1LTMuMDgybS0xNDAuODcyIDQ0LjM5Yy0xLjc3MS4wMTktMy4yNzMtMS40NjMtMy4yNy0zLjIyNXMxLjUxMS0zLjI2NCAzLjI1Ny0zLjI1MWMxLjcxNC4wMTMgMy4yMDIgMS40NzggMy4yNDQgMy4xOTIuMDQyIDEuNzI4LTEuNDcgMy4yNjUtMy4yMzEgMy4yODNtMzcuNjU2LTY1LjQ0MmMtMS43ODcuMDA4LTMuMjcxLTEuNDYxLTMuMjYxLTMuMjMuMDEtMS43NjMgMS41MTktMy4yNjcgMy4yNjItMy4yNTEgMS43MTUuMDE2IDMuMTk2IDEuNDkgMy4yMiAzLjIwNGEzLjI4IDMuMjggMCAwIDEtMy4yMjEgMy4yNzciLz48cGF0aCBkPSJNMTkxLjMxNCA4Mi44OGEzLjI3IDMuMjcgMCAwIDEtMy4yNDUgMy4yNTRjLTEuNzU1LS4wMDEtMy4yNDYtMS41MDgtMy4yMzYtMy4yNy4wMDktMS43MDIgMS40ODYtMy4xOCAzLjIxMS0zLjIxMiAxLjc1OC0uMDMzIDMuMjY3IDEuNDU2IDMuMjcxIDMuMjI5TTg1LjA1NiAxMDcuNDY1YTMuMjQgMy4yNCAwIDAgMSAzLjE2OSAzLjIyM2MtLjAwMyAxLjc1NS0xLjUxMyAzLjIzNi0zLjI4NCAzLjIyMS0xLjc1Mi0uMDE1LTMuMjYzLTEuNTU3LTMuMjI0LTMuMjg5LjA0LTEuNzc5IDEuNTI5LTMuMTg2IDMuMzM5LTMuMTU1bTY1LjUwMSAxMTkuNjA2Yy0xLjc2Ni4wMTMtMy4yODYtMS40NzYtMy4yOTEtMy4yMjItLjAwNC0xLjcyNSAxLjU0NS0zLjI2NSAzLjI3Ni0zLjI1MyAxLjcyMi4wMTEgMy4xODggMS40NjQgMy4yMjIgMy4xOTIuMDM0IDEuNzYyLTEuNDQgMy4yNzEtMy4yMDcgMy4yODQiLz48L2c+PHBhdGggZD0iTTE1NC45MjQgMTY2LjIzNGMtLjAwOC0uNTgtLjEzNS0xLjExNi42NzUtMS4wMy42NjEuMDcgMS4zMzUuMDg2IDEuMzMyLjk4Ni0uMDAzIDEuMDIyLS44MTYuODUtMS40NDguOTI4LS45NzYuMTIxLS4zNDktLjY3NS0uNTYtLjg4NG0tNy4wMTMgMS4zMzJsLjA3NS0xLjM1OGExLjA2IDEuMDYgMCAwIDEgMS4wNDktLjk2MWMuNTQ3LS4wMjcgMS4wMTMuMjQxIDEuMTUuODMxLjIyMi45NTUuMjMyIDEuOTIxLS4wMTQgMi44NzMtLjE2LjYyMS0uNjIxLjg3OS0xLjIzOS44MDlhMS4wMSAxLjAxIDAgMCAxLS45NzMtMS4wMTRjLS4wMy0uMzkxLS4wMDYtLjc4Ni0uMDA2LTEuMTc5bC0uMDQzLS4wMDEiIGNsYXNzPSJEIi8+PC9nPjwvc3ZnPg==
```

Comment:
the pre-commit check is complaining that there is no new line at the end of this file 😬

Reply:
lol I'll fix that